apollo-client/src/main/java/com/ctrip/framework/apollo/internals/AbstractConfig.java

```diff
@@ -84,6 +80,17 @@ public void addChangeListener(ConfigChangeListener listener, Set<String> interes
   public void addChangeListener(ConfigChangeListener listener, Set<String> interestedKeys, Set<String> interestedKeyPrefixes) {
     if (!m_listeners.contains(listener)) {
       m_listeners.add(listener);
+      Collections.sort(m_listeners, new Comparator<ConfigChangeListener>() {
+        @Override
+        public int compare(ConfigChangeListener o1, ConfigChangeListener o2) {
+          if (o1 instanceof ConfigOrderedChangeListener && o2 instanceof ConfigOrderedChangeListener){
```

Comment:
I think we can assume the order of ConfigChangeListener is default to 0 so that it can be compared with OrderedConfigChangeListener